socs/Lakeshore/Lakeshore372.py

```diff
@@ -1295,7 +1295,41 @@ def set_curve(self, _file):
             self._set_data_point(point[0], point[1], point[2])
 
         # refresh curve attributes
-        return self.get_header()
+        self.get_header()
+
+    def _check_curve(self, _file):
+        """After setting a data point for calibration curve,
+        use CRVPT? command from get_data_point() to check
+        that all points of calibration curve  were uploaded.
+        If not, re-upload points.
+
+        :param _file: calibration curve file
+        :type _file: str
+        """
+
+        with open(_file) as f:
+            content = f.readlines()
+
+        #skipping header info
+
+        values = []
+        for i in range(9, len(content)):
+            values.append(content[i].strip().split()) #data points that should have been uploaded
+
+
```

Comment:
Maybe this falls under personal preference here, but I'd remove one of these two blank lines. PEP08 recommends using "blank lines in functions, sparingly, to indicate logical sections."